BUILD.md

```diff
@@ -0,0 +1,9 @@
+To compile kubeztl:
+```powershell
```

Comment:
powershell??? :) all those commands look like they could just be 'shell' to me? I'll update and push back